metering/record.go

```diff
@@ -7,11 +7,11 @@ import (
 
 var logger = logrus.StandardLogger().WithField("metering", true)
 
-func RecordLogin(loginType string, userID, instanceID uuid.UUID) {
+func RecordLogin(loginType string, userID uuid.UUID) {
 	logger.WithFields(logrus.Fields{
 		"action":       "login",
 		"login_method": loginType,
-		"instance_id":  instanceID.String(),
+		"instance_id":  uuid.Nil.String(),
```

Comment:
Guessing we'll remove this at the next major release?

Reply:
Probably, not sure yet what it affects... but the value should have been 0-UUID if used without multi-instance mode.